app/src/main/java/org/mifos/mobile/ui/fragments/SettingsFragment.java

```diff
@@ -36,6 +36,7 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
     @Override
     public void onResume() {
         super.onResume();
+        getActivity().setTitle("Settings");
```

Comment:
Please remove hardcoded "Settings" strings from here. Using hardcoded strings is not a good practice. First, add the string in the `strigs.xml` file then use it dynamically.

Reply:
Yes. Will take care of it in future & will update it soon